packages/common/src/api/suggestedTracks.ts

```diff
@@ -172,20 +172,17 @@ export const useGetSuggestedPlaylistTracks = (collectionId: ID) => {
     isEqual
   )
 
-  useGetTracksByIds(
-    {
-      currentUserId,
-      ids: suggestedTracks
-        .filter(
-          (
-            suggestedTrack
-          ): suggestedTrack is { isLoading: true; id: ID; key: ID } =>
-            'id' in suggestedTrack && suggestedTrack.isLoading
-        )
-        .map((suggestedTrack) => suggestedTrack.id)
-    },
+  useTracks(
+    suggestedTracks
+      .filter(
+        (
+          suggestedTrack
+        ): suggestedTrack is { isLoading: true; id: ID; key: ID } =>
+          'id' in suggestedTrack && suggestedTrack.isLoading
+      )
+      .map((suggestedTrack) => suggestedTrack.id),
     {
-      disabled: !currentUserId || suggestedTrackIds.length === 0
+      enabled: !!currentUserId && suggestedTrackIds.length > 0
```

Comment:
heads up we dont need the suggestedTracks check since that is build into useTracks. if it's not you mind adding?